opbnb/addresses.yaml

```diff
@@ -0,0 +1,13 @@
+domainRoutingIsmFactory: "0x1AA7e1328BC28f585853A8d62F4cF0eB43ABAe8c"

```

Comment:
You new files should be under the `chains` folder

Reply:
done,